base_report_auto_create_qweb/models/report_xml.py

```diff
@@ -9,6 +9,12 @@
 class IrActionsReport(models.Model):
     _inherit = 'ir.actions.report.xml'
 
+    def _format_template_name(self, text):
```

Comment:
You can use slugify library (already a requirement) for doing this: https://github.com/odoo/odoo/blob/8.0/addons/website/models/website.py#L87